Makefile

```diff
@@ -15,15 +26,26 @@ base-docker-run = docker run \
 	-e AZURE_SUBSCRIPTION_ID=$(AZURE_SUBSCRIPTION_ID) \
 	-e AZURE_TENANT=$(AZURE_TENANT_ID) \
 	-e AZURE_TENANT_ID=$(AZURE_TENANT_ID) \
-	-v $(shell pwd):/packer-images
+	-v $(shell pwd):/packer-images \
+	$(docker_ssh_opts) \
+
+guard-%:
+	@ if [ "${${*}}" = "" ]; then \
+		echo "Variable '$*' not set"; \
+		exit 1; \
+	fi
 
 terraform-docker-run = $(base-docker-run) \
-	-w  /packer-images/terraform \
-	-it default-image-azure
+	-w  /packer-images/$(provider-dir) \
+	-it packer-images
+
+terraform-docker-run = $(base-docker-run) \
```

Comment:
This `terraform-docker-run` is duplicated

Reply:
Thanks!